SlevomatCodingStandard/Sniffs/Whitespaces/SpaceInNamedArgumentSniff.php

```diff
@@ -0,0 +1,67 @@
+<?php declare(strict_types = 1);
+
+namespace SlevomatCodingStandard\Sniffs\Whitespaces;
+
+use PHP_CodeSniffer\Files\File;
+use PHP_CodeSniffer\Sniffs\Sniff;
+use function sprintf;
+use const T_COLON;
+use const T_PARAM_NAME;
+use const T_WHITESPACE;
+
+class SpaceInNamedArgumentSniff implements Sniff
+{
+
+	public const ERROR_CODE = 'MissingSpaceAfterColon';
```

Comment:
There should be two different codes because there are two different types of error.